crates/dprint/src/arg_parser.rs

```diff
@@ -537,10 +537,12 @@ EXAMPLES:
     )
     .subcommand(
       Command::new("editor-info")
+        .about("Deprecated. Use lsp subcommand instead.")
         .hide(true)
     )
     .subcommand(
       Command::new("editor-service")
+        .about("Deprecated. Use lsp subcommand instead.")
```

Comment:
These are actually not deprecated yet. Lots of people are still using them and it's still the default in the vscode extension (because I haven't gotten around to finishing adding support for auto-completions for the config)